cloudfoundry-client-spring/src/test/java/org/cloudfoundry/client/spring/v2/serviceusageevents/SpringServiceUsageEventsTest.java

```diff
@@ -108,25 +111,25 @@ protected ListServiceUsageEventsResponse getResponse() {
                 .totalPages(1)
                 .totalResults(1)
                 .resource(ServiceUsageEventResource.builder()
-                    .metadata(Resource.Metadata.builder()
-                        .createdAt("2015-07-27T22:43:30Z")
-                        .id("0c9c59b8-3462-4acf-be39-aa987f087146")
-                        .url("/v2/service_usage_events/0c9c59b8-3462-4acf-be39-aa987f087146")
-                        .build())
-                    .entity(ServiceUsageEventsEntity.builder()
-                        .state("CREATED")
-                        .organizationId("guid-4dd5a051-3460-4246-a842-1dc2d5983c51")
-                        .spaceId("guid-76bd662b-fd5b-4b5c-a393-d65e67f99d53")
-                        .spaceName("name-2154")
-                        .serviceInstanceId("guid-15a7c119-838d-4516-acd9-062dec25d934")
-                        .serviceInstanceName("name-2155")
-                        .serviceInstanceType("type-2")
-                        .servicePlanId("guid-eddab64c-7be0-407e-91b0-82a8093cdfc5")
-                        .servicePlanName("name-2156")
-                        .serviceId("guid-d471c693-824c-44a6-b069-a679e323326d")
-                        .serviceLabel("label-77")
-                        .build())
-                    .build()
+                        .metadata(Resource.Metadata.builder()
```

Comment:
I'm not sure what's causing this formatting fluctuation, but you might want to double check that your project is using our formatting settings.  You'll want to ensure that you're using the "Project" scheme and that "Continuation indent" is at 4.

<img width="466" alt="screen shot 2016-02-18 at 06 57 23" src="https://cloud.githubusercontent.com/assets/60754/13147135/e6ba0490-d60c-11e5-8438-5a3dcf736ca3.png">

I'll fix in following commit.